Source/WebCore/dom/Element.cpp

```diff
@@ -2000,7 +2001,31 @@ ExceptionOr<bool> Element::toggleAttribute(const AtomString& qualifiedName, std:
     return true;
 }
 
-ExceptionOr<void> Element::setAttribute(const AtomString& qualifiedName, const AtomString& value)
+static ExceptionOr<String> getTrustedTypesCompliantAttributeValue(const String& attributeType, const TrustedTypeOrString& value, Element* element, String sink)
```

Comment:
"get" prefix should only be used for a function with an out argument.
This should be named `trustedTypesCompliantAttributeValue` instead.